pkgs/development/libraries/libxml2/default.nix

```diff
@@ -32,7 +33,7 @@ in
   assert oldVer -> stdenv.isDarwin; # reduce likelihood of using old libxml2 unintentionally
 
 let
-libxml = stdenv.mkDerivation rec {
+libxml = stdenv.mkDerivation (finalAttrs: rec {
```

Comment:
This `rec` I have kept because libxml package does something with `passthru` which I don't understand, and I don't want to break anything.